extensions/mssql/src/features.ts

```diff
@@ -30,6 +32,68 @@ export class TelemetryFeature implements StaticFeature {
 	}
 }
 
+export class NativeCredentialsFeature extends SqlOpsFeature<any>  {
```

Comment:
not sure `Native` is best name here (there term is a bit overloaded).  Maybe something like `KeytarCredentialsFeature` or `InternalCredentialsFeature` or something like that.